packages/test/test-utils/src/index.ts

```diff
@@ -3,6 +3,12 @@
  * Licensed under the MIT License.
  */
 
+export {
```

Comment:
Even if we do duplicate these here for use within test-utils, do we actually have to export them from the package too?  Or can we hide them for internal use only?

Reply:
There are other places in the repo that use these types as test details. Those usages would have to back to leveraging aqueduct directly. We can do that, if preferred. But since these exports are `@internal`, we can also freely remove them in the future as we clean up usages.

I could also mark these new exports as `@deprecated` if we want to discourage further use.